docs/how-it-works/identities/roles.md

```diff
@@ -0,0 +1,137 @@
+    assumed by an IAM user named John in AWS account 111111111111:
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": "sts:AssumeRole",
+          "Principal": {"AWS": "arn:aws:iam::111111111111:user/John"}
+        }
+      ]
+    }
+    ``` 
+    Note that a trust policy alone does NOT automatically give John permissions to assume this IAM role. 
+    Cross-account access always requires permissions in both accounts (2 way authorization). So, if John is in AWS account
+    111111111111 and you want him to have access to an IAM role called `DevOps` in account B ID 222222222222, then you need
+    to configure permissions in both accounts: 
+    1. In account 222222222222, the `DevOps` IAM role must have a trust policy that gives `sts:AssumeRole` permissions to 
+    AWS account A ID 111111111111 (as shown above).
+    2. 2nd, in account A 111111111111, you also need to attach an IAM policy to John’s IAM user that allows him to assume 
+    the `DevOps` IAM role, which might look like this:
+    
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+        {
+          "Effect": "Allow",
+          "Action": "sts:AssumeRole",
+          "Resource": "arn:aws:iam::222222222222:role/DevOps"
+        }
+      ]
+    }
+    ```
+
+## Assuming an AWS IAM role
+
+!!! summary "How does it work?" 
+    IAM roles do not have a user name, password, or permanent access keys. To use an IAM role, you must assume it by 
+    making an `AssumeRole` API call (vía [SDKs API](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html), 
+    [CLI](https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html) or 
+    [Web Console](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-console.html), which will
+    return temporary access keys you can use in follow-up API calls to authenticate as the IAM role. The temporary 
+    access keys will be valid for 1-12 hours (depending on your current validity expiration config), after which you
+    must call `AssumeRole` again to fetch new temporary keys. Note that to make the `AssumeRole` API call, you must
+    first authenticate to AWS using some other mechanism. 
+
+For example, for an IAM user to assume an IAM role, the workflow looks like this:
+![leverage-aws-iam-roles](../../assets/images/diagrams/aws-iam-role-assume.png "Leverage"){: style="width:900px"}
+
+<figcaption style="font-size:15px">
+<b>Figure:</b> Assuming an AWS IAM role.
+(Source: Gruntwork.io, 
+<a href="https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/#iam-roles">
+"How to configure a production-grade AWS account structure using Gruntwork AWS Landing Zone"</a>,
+Gruntwork.io Production deployment guides, accessed November 17th 2020).
+</figcaption>
+
+!!! example "Basic AssumoRole workflow"
```

Comment:
Small typo: "AssumoRole"  😉

Reply:
@diego-ojeda-binbash OJ thanks! I'll correct it before merging ✅

Reply:
@exequielrafaela - BBL-445 | fixing typo - d04310e